spec.bs

```diff
@@ -219,6 +220,12 @@ problems.
         window such that the distance between the top and bottom edges of the
         viewport are |options|["{{DocumentPictureInPictureOptions/height}}"]
         pixels.
+<p class="note">
```

Comment:
I believe this should be a normative "may" statement, instead of a non-normative "note".

(So, just remove the `<p class="note">` wrapper.)

Reply:
thanks, sgtm.  done.